doc/manual.md

```diff
@@ -2642,9 +2642,24 @@ of the argument.
 6. Conversion match: `a` is convertible to `f`, possibly via a user
    defined `converter`.
 
-These matching categories have a priority: An exact match is better than a
-literal match and that is better than a generic match etc. In the following,
-`count(p, m)` counts the number of matches of the matching category `m`
+It is important to note that overload resolution concerns itself with the 
+category of each parameter and not the category of callable.
+
+Nim will compare two candidates at a time and pick the "best" candidate to 
+to continue through the resolution process, if at the end the best candidate is 
+is proven "better" then the rest, it is chosen as an unambiguous match. I may help 
+to think of the below as a comparison between two candidates because of this algorithm. 
```

Comment:
It's irrelevant that "Nim will compare" (why the future here?) and it's irrelevant if it "compares two candidates at a time", it could compare them all at once too for that matter. You need to focus on the essential.